pkg/syncer/k8scloudoperator/placement.go

```diff
@@ -57,12 +63,17 @@ const (
 	// Resource name to get storage class.
 	resourceName = "storagepools"
 	// StoragePool type name for vsan-direct.
-	vsanDirect       = "vsanD"
-	invalidParamsErr = "FAILED_PLACEMENT-InvalidParams"
-	genericErr       = "FAILED_PLACEMENT-Generic"
-	notEnoughResErr  = "FAILED_PLACEMENT-NotEnoughResources"
-	invalidConfigErr = "FAILED_PLACEMENT-InvalidConfiguration"
-	vsanSna          = "vsan-sna"
+	vsanDirect                = "vsanD"
+	invalidParamsErr          = "FAILED_PLACEMENT-InvalidParams"
+	genericErr                = "FAILED_PLACEMENT-Generic"
+	notEnoughResErr           = "FAILED_PLACEMENT-NotEnoughResources"
+	invalidConfigErr          = "FAILED_PLACEMENT-InvalidConfiguration"
+	siblingReplicaBoundPVCErr = "FAILED_PLACEMENT-HasSiblingReplicaBoundPVC"
+	vsanSna                   = "vsan-sna"
+	// Appplatform label that all vDPP PVCs must have.
+	appplatformLabel = "appplatform.vmware.com/instance-id"
+	// Opt out label present on pod to look for bound PVCs of all sibling replicas
+	siblingReplicaBoundPVCCheckOptOutLabel = "psp.vmware.com/sibling-replica-bound-PVC-check-opt-out"
```

Comment:
suggestion: can we change this to something that partner understands, partners wouldn't know if we are checking whether bound PVCs or something else to find homeless pod. For them, we are checking for one replica snatching node of another replica. How about `sibling-replica-node-snatching-check-opt-out`?

Reply:
Yes that does sound better. I have updated the opt out label name.